module/proto/gravity/v1/query.proto

```diff
@@ -2,237 +2,271 @@ syntax = "proto3";
   }
-  rpc LastPendingValsetRequestByAddr(QueryLastPendingValsetRequestByAddrRequest) returns (QueryLastPendingValsetRequestByAddrResponse) {
-    option (google.api.http).get = "/gravity/v1beta/valset/last";
+  rpc ContractCallTxs(ContractCallTxsRequest) returns (ContractCallTxsResponse) {
+    option (google.api.http).get = "/gravity/v1/batch/logic_calls";
   }
-  rpc LastPendingBatchRequestByAddr(QueryLastPendingBatchRequestByAddrRequest) returns (QueryLastPendingBatchRequestByAddrResponse) {
-    option (google.api.http).get = "/gravity/v1beta/batch/{address}";
+
+  // ethereum signature queries so validators can construct valid etherum transactions
+  // TODO: can/should we group these into one endpoint?
+  rpc SignerSetEthereumSignatures(SignerSetEthereumSignaturesRequest) returns (SignerSetEthereumSignaturesResponse) {
+    option (google.api.http).get = "/gravity/v1/signer_sets/EthereumSignatures";
   }
-  rpc LastPendingLogicCallByAddr(QueryLastPendingLogicCallByAddrRequest) returns (QueryLastPendingLogicCallByAddrResponse) {
-    option (google.api.http).get = "/gravity/v1beta/logic/{address}";
+  rpc BatchTxEthereumSignatures(BatchTxEthereumSignaturesRequest) returns (BatchTxEthereumSignaturesResponse) {
+    option (google.api.http).get = "/gravity/v1/batch_txs/EthereumSignatures";
   }
-  rpc LastEventNonceByAddr(QueryLastEventNonceByAddrRequest) returns (QueryLastEventNonceByAddrResponse) {
-    option (google.api.http).get = "/gravity/v1beta/oracle/eventnonce/{address}";
+  rpc ContractCallTxEthereumSignatures(ContractCallTxEthereumSignaturesRequest) returns (ContractCallTxEthereumSignaturesResponse) {
+    option (google.api.http).get = "/gravity/v1/logic_calls/EthereumSignatures";
   }
-  rpc BatchFees(QueryBatchFeeRequest) returns (QueryBatchFeeResponse) {
-    option (google.api.http).get = "/gravity/v1beta/batchfees";
+  
+  // pending ethereum signature queries for orchestrators to figure out which signatures they are missing
+  // TODO: can/should we group this into one endpoint?
+  rpc PendingSignerSetEthereumSignatures(PendingSignerSetEthereumSignaturesRequest) returns (PendingSignerSetEthereumSignaturesResponse) {
+    option (google.api.http).get = "/gravity/v1/signersets/{address}/pending";
   }
-
-  rpc OutgoingTxBatches(QueryOutgoingTxBatchesRequest) returns (QueryOutgoingTxBatchesResponse) {
-    option (google.api.http).get = "/gravity/v1beta/batch/outgoingtx";
+  rpc PendingBatchTxEthereumSignatures(PendingBatchTxEthereumSignaturesRequest) returns (PendingBatchTxEthereumSignaturesResponse) {
+    option (google.api.http).get = "/gravity/v1/batches/{address}/pending";
   }
-  rpc OutgoingLogicCalls(QueryOutgoingLogicCallsRequest) returns (QueryOutgoingLogicCallsResponse) {
-    option (google.api.http).get = "/gravity/v1beta/batch/outgoinglogic";
+  rpc PendingContractCallTxEthereumSignatures(PendingContractCallTxEthereumSignaturesRequest) returns (PendingContractCallTxEthereumSignaturesResponse) {
```

Comment:
I don't understand the use of the word "pending" here.

Reply:
Good catch. This is maybe `unsigned` instead? This is supposed to return the ones that a validator hasn't yet signed.